src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -106,7 +114,7 @@ public static Configuration mapJdbcCatalogToConfig(Configuration ingestConfig, S
 
     public static void writeConfigToYaml(Configuration outputConfig, String outputDir, String fileName) throws IOException {
         if (outputDir == null || outputDir.equals("")) {
-            outputDir = outputConfig.getName() + "/" + outputConfig.getEnvironment() + "/conf";
+            outputDir = "output/" + outputConfig.getName() + "/" + outputConfig.getEnvironment() + "/conf";
```

Comment:
String.format would clean this up nicely

Reply:
updated